testsuite/musictests/test_event_proxies_receiver.sli

```diff
@@ -32,7 +32,7 @@ meip << /port_name (spikes_in) /music_channel 0 >> SetStatus
 meip n << /weight 750.0 >> Connect
```

Comment:
This comment applies to the MPI-check a little further up. Could you replace the explicit `quit_i` code with an `exit_test_gracefully` (or what it is called exactly)?